versions/TENSORFLOW_C_VERSION

```diff
@@ -1 +1 @@
-1.15.0
+2.3.0
```

Comment:
I thought we have a problem with tf v2.0.0~2.2.0, can we use v2.3.0 ? If we can use,it should be v2.3.1

Reply:
@kpango there's no problem. @datelier confirmed that v2.3.0 is okay.

there's no prebuilt lib of version v2.3.1 here, so it should be v2.3.0.
https://www.tensorflow.org/install/lang_c

Reply:
okay, I understand LGTM

Reply:
thanks